templates/java/org/prism/Loader.java.erb

```diff
@@ -54,6 +54,8 @@ public class Loader {
 
             if (constant == null) {
                 int offset = bufferOffset + index * 8;
+
+                // Read an integer in forced little endian.
```

Comment:
I would remove this, that should be clear from the `this.buffer = ByteBuffer.wrap(serialized).order(ByteOrder.LITTLE_ENDIAN);` line below.